dbterd/cli/params.py

```diff
@@ -63,6 +63,13 @@ def common_params(func):
         show_default=True,
         type=click.STRING,
     )
+    @click.option(
+        "--omit_columns",
```

Comment:
Thanks @syou6162 great PR again!

Can we use hyphen here instead of underscore for consistency?

Reply:
Also if possible, can you help to add doc of the CLI reference in https://github.com/datnguye/dbterd/blob/main/docs/nav/guide/cli-references.md for this new option? Thanks a lot

Reply:
@datnguye Thank you for your review!

- I fixed option name in 540c56c
- I added docs for new option in b172040